scripts/release_info.py

```diff
@@ -54,11 +54,6 @@
                     'CURRENT_COLLECTION_SCHEMA_VERSION']
 FIRST_OPPIA_COMMIT = '6a7138f5f603375e58d1dc3e1c4f1c80a126e249'
```

Comment:
I think it is fine to have them since they are specific to this file only. I added only common constants through multiple files in release constants file. What do you think?